### DISCUSSION TITLE
Welcome to bike-traffic-plotly-dash Discussions!

### DISCUSSION THREAD

**Discussion by frfeng** [fenggroup/bike-traffic-plotly-dash#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vielmetti** [comment]
Hi! I'm a regular bicycle commuter to downtown Ann Arbor (coming in to Main St via Packard or Forest). Interested in civic data in general as well, and how it can be used to support decision making.

**Reply by frfeng** [reply]
Welcome, Edward! In addition to GitHub, we also just set up this Twitter account <https://twitter.com/a2bikecounters> to share non-technical updates on this project.

**Comment by vielmetti** [comment]
Thanks @frfeng ! You're welcome also if you need a Mastodon spot to post to https://a2mi.social
